packages/go_router/lib/src/route.dart

```diff
@@ -269,6 +269,7 @@ class GoRoute extends RouteBase {
     super.redirect,
     this.onExit,
     super.routes = const <RouteBase>[],
+    this.fragment,
```

Comment:
why is this needed?

Reply:
You are right , I have removed the fragment part from the file!. Sorry for the inconvenience and Thanks for supporting